src/main/java/org/elasticsearch/cluster/block/ClusterBlockLevel.java

```diff
@@ -29,9 +29,18 @@
 public enum ClusterBlockLevel {
     READ(0),
     WRITE(1),
-    METADATA(2);
 
-    public static final EnumSet<ClusterBlockLevel> ALL = EnumSet.of(READ, WRITE, METADATA);
+    /**
+     * After 1.5.0 METADATA has been splitted into two distincts cluster block levels
```

Comment:
I would be more precise on the version, cause it's not clear if it is from 1.5.1 or 1.6.0.